packages/core/src/client.ts

```diff
@@ -2628,6 +2633,16 @@ export class MedplumClient extends EventTarget {
       // Silently ignore if this environment does not support storage events
     }
   }
+
+  private retryCatch(retryNumber: number, maxRetries: number, err: Error): void {
+    // This is for the 1st retry to avoid multiple notifications
+    if (retryNumber === 1) {
```

Comment:
This is to show just 1 notification despite the retry

Reply:
It looks like `retryNumber` is actually zero the first time this would be called on failure; should that be the condition instead?

Reply:
I'm still investigating this, but if I put the condition as 0, it doesn't dispatch, but it dispatches at 1 and 2.